node/src/neighborhood/mod.rs

```diff
@@ -2076,7 +2078,8 @@ pub enum FallbackPreference {
 // fallback_preference is enum, that controls whether we want to strictly prohibit exit_location to nodes
 // with requested country_codes, or we accept other locations in case requested country is unavailable
 // locations_opt is Optional Vec of ExitLocation, it is set to Some(Vec<ExitLocation>) from users input,
-// where ExitLocation is set of countries with same priority
+// where ExitLocation is a set of countries with the same priority. None is state, when user did not set any
```

Comment:
How about, "locations_opt will be `None` when the user did not set any country for exit at all."?